cypress/e2e/app_tests/test_3.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('Form layout page', () => {
+    it('Type data in Horizontal form', () => {
+        cy.visit('/pages/forms/layouts');
+        cy.get('.form-horizontal .col-sm-9 .input-full-width.size-medium.shape-rectangle[type="email"]').clear().type('test@email.com')
```

Comment:
You can write your selector shorter :
.form-horizontal [type="email"]

Reply:
Updated. Thanks
Now is clear. I was thinking that the more precise selector is better.